manifests/b/Beowulfso/BSU/0.0.52/Beowulfso.BSU.locale.en-GB.yaml

```diff
@@ -0,0 +1,24 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: Beowulfso.BSU
+PackageVersion: 0.0.52
+PackageLocale: en-GB
+Publisher: Beowulf Strategic Operations
+PublisherUrl: https://u.beowulfso.com/
+PublisherSupportUrl: https://u.beowulfso.com/
+#PrivacyUrl: 
+#Author: 
+PackageName: Beowulf Sync Utility (Prototype)
+PackageUrl: https://u.beowulfso.com/
+License: MIT
+LicenseUrl: https://github.com/BeowulfStratOps/BSU.Sync/blob/master/LICENSE.txt
+#Copyright: 
+#CopyrightUrl: 
+ShortDescription: Mod Downloader and Updater for Arma 3
+#Description: 
+Moniker: bsu
+Tags:
+- arma bsu beowulf beowulfso

```

Comment:
```suggestion
- arma
- bsu
- beowulf
- beowulfso
```

Reply:
I'll commit the suggestion as it looks neater.

This is how the YamlCreate.PS1 file generated the YAML though with it all on 1 line.